### PR TITLE
Update my ppp

### PR DESCRIPTION
**Pull request by kangzongxian**

### CODE REVIEW

File: docs/team/kangzongxian.md [hunk 1]
```diff
@@ -5,36 +5,55 @@ title: Kang Zong Xian's Project Portfolio Page
...
 
     * Developer Guide:
-        * to be added soon
+        * Did final checks to ensure consistency of DG (pull request #140).
+        * Added implementation of find command in DG (pull request #114).
+        * Add UML Diagrams (pull request #114)
 
 * **Contributions to team-based tasks**:
-    * to be added soon
+    * Facilitated and tidied up contents of postmortem v1.2
```
Comment (Lan-Jingbo) [inline docs/team/kangzongxian.md:46]:
```suggestion
    * Facilitated and tidied up contents of postmortem v1.2
    * Facilitated and tidied up contents of postmortem v1.3
    * Facilitated and tidied up contents of postmortem v1.4
```

File: docs/team/kangzongxian.md [hunk 2]
```diff
@@ -5,36 +5,55 @@ title: Kang Zong Xian's Project Portfolio Page
...
+
+**Project management**:
+* Used Github issue tracker.
+* Followed the branching workflow.
+* Contributed to meetings by taking down minutes and facilitating discussions.
 
 * **Review/mentoring contributions**:
-    * to be added soon
+  * Team members' PR reviews (with non-trivial review comments): pull request #191, #142, #138, #131, #119, #111
+    , #109, #73, #68, #65, #58
```
Comment (Lan-Jingbo) [inline docs/team/kangzongxian.md:55]:
```suggestion
  * Team members' PR reviews (with non-trivial review comments): pull request #194, #193, #152, #147, #146, #145, #140, #137, #132, #130, #124, #114, #102, #79, #74, #70, #55, #54
```

**Review by Lan-Jingbo — CHANGES_REQUESTED** [review]

File: docs/team/kangzongxian.md (hunk 1 above)
Comment (kangzongxian) [reply docs/team/kangzongxian.md:46]: There was only postmortem for v1.2 so I think just v1.2 is fine!

File: docs/team/kangzongxian.md (hunk 2 above)
Comment (kangzongxian) [reply docs/team/kangzongxian.md:55]: I think the PR's you mention are the ones I created and not the ones I reviewed, which I think is what the statement wants!

**Review by Lan-Jingbo — APPROVED** [review]

**Pull request merged**